core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -141,19 +146,26 @@ export class AudioBarComponent {
     this.autogeneratedAudioPlayerService.cancel();
   }
 
-  getCurrentAudioLanguageCode(): string {
+  // Returns null if the audio is not available in the current language.
+  getCurrentAudioLanguageCode(): string | null {
     return this.audioTranslationLanguageService
       .getCurrentAudioLanguageCode();
   }
 
-  getCurrentAudioLanguageDescription(): string {
+  // Returns null if the language is not available.
```

Comment:
What does it mean that the language is not available?

Reply:
Done